taps-api/data/venues.js

```diff
@@ -28,19 +28,22 @@ module.exports = {
     if (typeof lng !== 'string' || lng.trim() === '') throw new Error('lng must be a non-empty string');
```

Comment:
When parsing numbers make sure they are between the valid coordinates that they can be according to mongo website.

Reply:
https://docs.mongodb.com/manual/reference/geojson/